wasm/Cargo.toml

```diff
@@ -16,7 +16,8 @@ members = [
 [profile.dev]
 panic = "abort"
 
-[profile.release]
+[profile.deploy]
+inherits = "release"
```

Comment:
I support this change. But we need to do something about building test executor. It shouldn't have stripped `debuginfo`. We want stack traces